ios/RCTWebRTC/WebRTCModule.m

```diff
@@ -29,20 +30,29 @@ - (instancetype)init
 //    [RTCPeerConnectionFactory initializeSSL];
 
     _peerConnections = [NSMutableDictionary new];
-    _remoteStreams = [NSMutableDictionary new];
-    _remoteTracks = [NSMutableDictionary new];
     _localStreams = [NSMutableDictionary new];
     _localTracks = [NSMutableDictionary new];
   }
   return self;
 }
 
+- (RTCMediaStream*)streamForTag:(NSString*)reactTag
+{
+    RTCMediaStream *stream = _localStreams[reactTag];
+    if (!stream) {
+        for (NSNumber *peerConnectionId in _peerConnections) {
```

Comment:
I don't see any easier way as far as my Objective C and stackoverflow knowledge goes...